vpc.auto.tfvars.example

```diff
@@ -0,0 +1,26 @@
+## Required values
+name = "" # Type: string. Name of your vpc. (e.g. `app` or `cluster`)
+
+## Geodessic module inherrited values
+namespace = "" #Type: string. Namespace (e.g. `cp` or `cloudposse`)
```

Comment:
This should typically not be defined at the module level, but rather come from the environment.